CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,45 @@
-# Code of conduct
+# Community Code of Conduct
 
-Each contributor and maintainer of this project agrees to follow the [community Code of Conduct](https://github.com/kyma-project/community/blob/master/CODE_OF_CONDUCT.md) that relies on the CNCF Code of Conduct. Read it to learn about the agreed standards of behavior, shared values that govern our community, and details on how to report any suspected Code of Conduct violations.
+## Overview
+
+This document makes a reference to the rules of behavior for all contributors and maintainers in the community. All members agree to follow them to foster the growth of the community of tolerance, respect, and mutual understanding.
+
+## Code of conduct
```

Comment:
I was thinking - if the code of conduct is taken from https://www.contributor-covenant.org/version/1/2/0/code-of-conduct, maybe we can simply link to it?  Or have something shorter like here in community: https://github.com/kyma-project/community/blob/master/CODE_OF_CONDUCT.md#code-of-conduct with an additional link?

Reply:
[Here](https://github.com/SAP/luigi/pull/531/commits/383e3f73b905348429ef0541282816d50025525b) my attempt, please check.

Reply:
Yes, this is fine 👍 :) I think keeping it shorter does the job, and the link gives more info if needed.